### ISSUE TITLE
about Quantization-aware training files in https://www.yuque.com/mnn/en/ricgpz you provided

### ISSUE THREAD

**Issue by dandanli814** [alibaba/MNN#868]
Hi, than you for your job.
When I run the files of quantization-aware training from https://www.yuque.com/mnn/en/ricgpz, I want to know the test accuracy of the model corresponding to this file "mobilenet_v2_tfpb_train_public.mnn" you provided. I want to compare this accuracy with the quantized model accuracy.
Looking forward to your answer.

**Comment by yyfcc17** [comment]
this is a standard mobilenetV2 model converted from tf official frozen pb. our test accuracy is presented here: https://www.yuque.com/mnn/en/tqr7ft

**Comment by dandanli814** [comment]
Thank you. I see it is 72.324% right? And I run the the demo you provided in pymnn/examples/MNNTrain/quantization_aware_training, the final result is as following:
![image](https://user-images.githubusercontent.com/24379250/83854479-5b092a80-a749-11ea-9653-bf585f8faba8.png)

The accuracy before and after quantization seems to have dropped by about two points. But you provide is 72.456% as following:
![image](https://user-images.githubusercontent.com/24379250/83854868-e4206180-a749-11ea-9d82-aae4d9d95a1e.png)

I don't know why, can you help me?

**Comment by yyfcc17** [comment]
the result in the table is a standard experimental result, which is obtained by training on the imagenet training dataset and test on the validation dataset(50000 images), using MNN c++ api.
but in the demo, the training dataset and test dataset is divided from validation dataset, and using python api.
so there will be some minor differences between them.

consider the fact that not everyone has full imagenet training dataset, so we use validation dataset to finetune. we wrote this demo to show how to use the QAT interface.

so if you want to reproduce the result, you may need to get the full imagenet2012 training dataset.

**Comment by dandanli814** [comment]
OK, thank you. ^_^
